### PR TITLE
[virtualization] Minor doc updates

### PR DESCRIPTION
**Pull request by z9r5**
## Description
Minor updates

## Checklist
- [ ] The code is covered by unit tests.
- [ ] e2e tests passed.
- [ ] Documentation updated according to the changes.
- [ ] Changes were tested in the Kubernetes cluster manually.

## Changelog entries
```changes
section: linstor
type: chore
summary: Minor doc updates.
impact_level: low
```

### CODE REVIEW

File: modules/490-virtualization/docs/README_RU.md [hunk 1]
```diff
@@ -2,4 +2,10 @@
 title: "Модуль virtualization"
 ---
 
-Модуль предоставляет возможность запуска виртуальных машин.
+Модуль позволяет управлять виртуальными машинами с помощью Kubernetes. Модуль использует проект [kubevirt](https://github.com/kubevirt/kubevirt).
```
Comment (kirkonru) [inline modules/490-virtualization/docs/README_RU.md:5]:
```suggestion
Модуль позволяет управлять виртуальными машинами с помощью Kubernetes. Базируется на проекте [KubeVirt](https://github.com/kubevirt/kubevirt).
```

File: modules/490-virtualization/docs/README_RU.md [hunk 2]
```diff
@@ -2,4 +2,10 @@
...
+
+Для работы виртуальных машин используется стэк QEMU (KVM) + libvirtd и CNI Cilium (необходим включенный модуль [cni-cilium](../021-cni-cilium/)). В качестве хранилища гарантируется работа с [LINSTOR](../041-linstor) или [Ceph](../099-ceph-csi/), но также возможны и другие варианты хранилища.
```
Comment (kirkonru) [inline modules/490-virtualization/docs/README_RU.md:7]:
```suggestion
Для работы виртуальных машин используется стек QEMU (KVM) + libvirtd и CNI Cilium (необходим включенный модуль [cni-cilium](../021-cni-cilium/)). Модуль поддерживает работу с платформами хранения данных [LINSTOR](../041-linstor) или [Ceph](../099-ceph-csi/). Возможны и другие варианты хранилища.
```

File: modules/490-virtualization/docs/README_RU.md [hunk 3]
```diff
@@ -2,4 +2,10 @@
...
+
+Основные преимущества модуля:
+- Простой интерфейс работы с виртуальными машинами как [примитивами Kubernetes](cr.html) (работа с ВМ аналогична работе с Pod'ами);
```
Comment (kirkonru) [inline modules/490-virtualization/docs/README_RU.md:10]:
```suggestion
- Простой и понятный интерфейс для работы с виртуальными машинами как с [примитивами Kubernetes](cr.html) (работать с виртуальными машинами так же легко, как с Pod'ами);
```

File: modules/490-virtualization/docs/README_RU.md [hunk 4]
```diff
@@ -2,4 +2,10 @@
...
+- Высокая производительность сетевого взаимодействия за счет использования CNI cilium с поддержкой [MacVTap](https://github.com/kvaps/community/blob/macvtap-mode-for-pod-networking/design-proposals/macvtap-mode-for-pod-networking/macvtap-mode-for-pod-networking.md) (исключает накладные расходы на трансляцию адресов).
```
Comment (kirkonru) [inline modules/490-virtualization/docs/README_RU.md:11]:
```suggestion
- Высокая производительность сетевого взаимодействия за счет использования CNI Сilium с поддержкой [MacVTap](https://github.com/kvaps/community/blob/macvtap-mode-for-pod-networking/design-proposals/macvtap-mode-for-pod-networking/macvtap-mode-for-pod-networking.md) (позволяет сэкономить вычислительные ресурсы, которые в противном случае расходовались бы на трансляцию адресов).
```

**Review by kirkonru — DISMISSED** [review]

File: modules/490-virtualization/docs/README_RU.md [hunk 5]
```diff
@@ -2,4 +2,10 @@
...
 ---
 
-Модуль предоставляет возможность запуска виртуальных машин.
+Модуль позволяет управлять виртуальными машинами с помощью Kubernetes. Базируется на проекте [KubeVirt](https://github.com/kubevirt/kubevirt).
+
+Для работы виртуальных машин используется стек QEMU (KVM) + libvirtd и CNI Cilium (необходим включенный модуль [cni-cilium](../021-cni-cilium/)). Модуль поддерживает работу с платформами хранения данных [LINSTOR](../041-linstor) или [Ceph](../099-ceph-csi/). Возможны и другие варианты хранилища.
+
+Основные преимущества модуля:
+- Простой и понятный интерфейс для работы с виртуальными машинами как с [примитивами Kubernetes](cr.html) (работать с виртуальными машинами так же легко, как с Pod'ами);
+- Высокая производительность сетевого взаимодействия за счет использования CNI Сilium с поддержкой [MacVTap](https://github.com/kvaps/community/blob/macvtap-mode-for-pod-networking/design-proposals/macvtap-mode-for-pod-networking/macvtap-mode-for-pod-networking.md) (позволяет сэкономить вычислительные ресурсы, которые в противном случае расходовались бы на трансляцию адресов).
```
Comment (z9r5) [inline modules/490-virtualization/docs/README_RU.md:11]:
```suggestion
- Высокая производительность сетевого взаимодействия за счет использования CNI Сilium с поддержкой [MacVTap](https://github.com/kvaps/community/blob/macvtap-mode-for-pod-networking/design-proposals/macvtap-mode-for-pod-networking/macvtap-mode-for-pod-networking.md) (исключает накладные расходы на трансляцию адресов).
```

**Comment by deckhouse-BOaTswain** [comment]
Cherry pick PR [3698](https://github.com/deckhouse/deckhouse/pull/3698) to the branch [release-1.43](https://github.com/deckhouse/deckhouse/tree/release-1.43) successful!

**Pull request merged**